go/common/logs.go

```diff
@@ -17,6 +17,8 @@ type LogSubscription struct {
 	Signature *[]byte
 	// A subscriber-defined filter to apply to the stream of logs.
 	Filter *filters.FilterCriteria
+	// Used to track which rollups the subscription has already been sent logs for.
+	LastSeenRollup uint64
```

Comment:
I wonder if this also needs to include a hash for the last seen rollup to make sure logs are updated if things change. Might be worth adding a todo, not sure if it's an issue or not but if it is I'll come across it with this block ordering work hopefully.

Reply:
Cool, added a general todo to think about chain reorgs. Thanks.